src/main/webapp/add-account.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Add File To Configuration</title>
+    <link rel="stylesheet" href="assets/add-account.css">
+    <link rel="stylesheet" href="assets/common.css">
+    <script src="scripts/appscript.js"></script>
+  </head>
+  <body onload="javascript:populateCustomerList()">
+    <div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="readfile.html">Read Configuration</a>
+        <a href="createfile.html">Create Configuration</a>        
```

Comment:
The indentation of this line and the 3 below looks wrong

Reply:
That is a GH problem. It looks correct on cloudshell.

Reply:
I've had this problem with HTML files as well and I think it is a problem with cloudshell. My version of this branch looks the same on intelliJ and vim